src/nodeconductor_test/private_settings.py.example

```diff
@@ -1,12 +1,12 @@
 class AWSPrivateSettings(object):
-    access_key_id_name = 'some_key_id'
-    secret_access_key_name = 'some_access_key'
+    aws_access_key_id = 'my_aws_access_key_id'
+    aws_secret_access_key = 'my_aws_access_key'
```

Comment:
Reason for value changes:

Automatic test runners will replace these placeholders with actual values, so the placeholders should be unique and easily detectable enough for replacements to work correctly.